internal/provider/provider.go

```diff
@@ -120,17 +124,17 @@ func (p *TimescaleProvider) Configure(ctx context.Context, req provider.Configur
 func (p *TimescaleProvider) Resources(ctx context.Context) []func() resource.Resource {
 	tflog.Trace(ctx, "TimescaleProvider.Resources")
 	return []func() resource.Resource{
-		NewServiceResource,
+		r_service.NewServiceResource,
 	}
 }
 
 // DataSources defines the data sources implemented in the provider.
 func (p *TimescaleProvider) DataSources(ctx context.Context) []func() datasource.DataSource {
 	tflog.Trace(ctx, "TimescaleProvider.DataSources")
 	return []func() datasource.DataSource{
-		NewProductsDataSource,
-		NewServiceDataSource,
-		NewVpcsDataSource,
+		d_products.NewProductsDataSource,
```

Comment:
now that everything is in packages we can remove the Timescale naming from everything. so if we stay with this it'd be `d_products.NewDataSource` or with the other suggestion `products.NewDataSource`